keyboards/ai03/polaris/keymaps/mekberg/config.h

```diff
@@ -0,0 +1,5 @@
+#pragma once
```

Comment:
Could you add a GPL2+ license header, here?

Reply:
Sure, like that? I copied over the one from the original Polaris files, so they're the same.

Reply:
Awesome! Thanks!
And you can add your name there too, if you want.